### ISSUE TITLE
/TH/NODE leads to an incorrect FRAME/SKEW influenced by /TH/PART

### ISSUE THREAD

**Issue by SonSiont** [OpenRadioss/OpenRadioss#1410]
**Describe the bug**
Node 100, DX =  0, DZ = 65 in TH of i4.k, wrong.
Node 100, DX = 58, DZ =  0 in TH of conrad_0000.rad, maybe correct.

**To Reproduce**
Run i4.k, the original LS-DYNA keyword file.
Run the homemade converted file, conrad_0000.rad .
Compare the DX/DZ of node 100 in both (.csv) results.

**Expected behavior**
The results should have a correct value in the correct FRAME/SKEW

**Output**
The input: 
[th.tar.gz](https://github.com/OpenRadioss/OpenRadioss/files/11342977/th.tar.gz)

**Desktop (please complete the following information):**
 - Version [d0ba914034306b89045b50d0954d078f9092b2ad]
- Hardware [Linux; Intel(R) Core(TM) i7-5930K CPU @ 3.50GHz (x86_64), 3498 MHz,  64217 MB RAM]
- Configuration [Double precision SMP, 4 tasks]

**Additional context**
When change the lines in conrad_0000.rad

```
1834,1835c1834,1835
<         IE        KE        HE       XCG       YCG       ZCG
< #wrong    ERODED      HEAT
---
> #OK        IE        KE        HE       XCG       YCG       ZCG
>     ERODED      HEAT
```

the result is now wrong. The reason seems to be the combination of "ERODED" and "HEAT".

**Comment by sebastienVilleneuve** [comment]
correction ongoing with https://github.com/OpenRadioss/Tools/pull/3

**Comment by sebastienVilleneuve** [comment]
solved with ca3d17c762ed6d705673096b763430794fd18246

Regards,
Sébastien
